src/Components/Blazor/Build/src/Core/BootJsonWriter.cs

```diff
@@ -40,7 +40,7 @@ public static string GetBootJsonContent(string assemblyFileName, string entryPoi
                 assemblyReferences,
                 embeddedContent,
                 linkerEnabled);
-            return Json.Serialize(data);
+            return JsonSerializer.ToString(data, new JsonSerializerOptions { PropertyNamingPolicy = JsonNamingPolicy.CamelCase });
```

Comment:
Worth making this a shared constant somewhere?

Reply:
User visible?

Reply:
I don't think it need to be.